docs/components/TopLevelNav/TopLevelNav.tsx

```diff
@@ -134,16 +139,28 @@ const TopLevelNav = React.forwardRef((props: TopLevelNavProps, ref: React.Ref<HT
     }
   };
 
+  const ArrowIcon = rtl ? (
+    showSubmenu ? (
+      <AngleRight />
+    ) : (
+      <AngleLeft />
+    )
+  ) : showSubmenu ? (
+    <AngleLeft />
+  ) : (
+    <AngleRight />
+  );
```

Comment:
Nesting ternary expressions can make code more difficult to understand.

https://eslint.org/docs/latest/rules/no-nested-ternary#rule-details